docker/Dockerfile.cpu

```diff
@@ -21,7 +21,7 @@ RUN apt-get update && \
     && cd /tmp \
     && curl -O https://bootstrap.pypa.io/get-pip.py \
     && python get-pip.py \
-    && pip install gevent gunicorn mxnet-model-server \
+    && pip install gevent gunicorn mxnet-model-server==0.3.0b1 \
```

Comment:
Why not --pre?
This ensures it will pull in newer beta versions as well

Reply:
I redid this. --pre seems to work fine.